Sources/StreamChat/APIClient/APIClient.swift

```diff
@@ -288,7 +293,32 @@ class APIClient {
         // We only retry transient errors like connectivity stuff or HTTP 5xx errors
         ClientError.isEphemeral(error: error)
     }
-
+    
+    func downloadAttachment(
+        _ attachment: ChatMessageFileAttachment,
```

Comment:
Why is this limited to file attachments? 🤔 We should be able to download any attachment?

Reply:
Fair point, as long as we have the URL to the file we should be able to download it.